src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyGroupRequest.java

```diff
@@ -105,7 +105,7 @@ void doRequest(
     boolean canDeferRequest(
         HttpCacheProxyCacheableRequest newRequest)
     {
-        return request.prefer == null || request.ifNoneMatch == null || !request.maxAgeZero ||
+        return request.prefer == null || request.ifNoneMatch == null || request.maxAgeZero ||
```

Comment:
I think this one is supposed to be `!` since it is saying if the request is not (prefer wait + if-none-match + max-age 0) so it will complete.

That means prefer could be null, or if-none-match could be null or max-age could be non-zero.

Reply:
thanks good catch I was supposed to revert it back and missed it.